src/frontend/src/components/authenticateBox.ts

```diff
@@ -565,16 +576,29 @@ export const authnScreens = (i18n: I18n, props: AuthnTemplates) => {
             resolve({ tag: "recover", userNumber }),
         })
       ),
-    pick: (pickProps: { anchors: NonEmptyArray<bigint> }) =>
+    pick: (pickProps: {
+      anchors: NonEmptyArray<bigint>;
+      autoSelect?: bigint;
+    }) =>
       new Promise<
         { tag: "more_options" } | { tag: "pick"; userNumber: bigint }
-      >((resolve) =>
+      >((resolve) => {
+        // render page first so that when the identity is picked and the passkey
```

Comment:
What is it that we need to wait? Couldn't we show a spinner while we wait for it and then resolve?

Reply:
Well, normally we wait for the user to chose. But if we have an a `autoSelect` identity, then the action completes immediately. I am just threading it through here, because the identity picker should still be rendered, before automatically selecting something. Otherwise you have the passkey interaction from a completely blank page, which is pretty weird. This way, you have at least a (albeit blurred out) version of the II page in the background.

Reply:
We could render something else if we want to improve it I guess. But definitely not a priority.